models/autoencoders/dvaepp.py

```diff
@@ -106,9 +106,9 @@ def kl_divergence(self, post_dists, post_samples, is_training=True):
         rbm_vis = rbm_visible_samples.detach()
         rbm_hid = rbm_hidden_samples.detach()
         
-        batch_energy = (torch.sum(torch.matmul(rbm_vis, torch.matmul(self.prior.get_weights(), rbm_hid.t())), 1) 
-                        + torch.matmul(rbm_vis, self.prior.get_visible_bias())
-                        + torch.matmul(rbm_hid, self.prior.get_hidden_bias()))
+        batch_energy = (torch.sum(torch.matmul(rbm_vis, torch.matmul(self.prior.weights, rbm_hid.t())), 1) 
```

Comment:
The getter-methods in the `RBM` class are now properties, that's why the syntax changed here.